### DISCUSSION TITLE
New name?

### DISCUSSION THREAD

**Discussion by bongobongoland** [Pennyw0rth/NetExec#8] [General]
Since the father of CME doesn't want us to use the name CrackMapExec, what name shall we use then?

KrackMapExec? 
CarCakeMex
PamCreamCake?

**Comment by bongobongoland** [comment]
I already have an idea for a new logo :) If we go with KME.

**Comment by noraj** [comment]
I'll repaste some random ideas I posted on https://github.com/Porchetta-Industries/CrackMapExec/discussions/801#discussioncomment-6957593

### Using CrackMapExec name

Ideal so that people coming from legacy CME are not confused and understand it's the continuation of the project or "new version".

###### Idea 1

- repo/package name: crackmapexec-ng
- bin short name cme-ng

###### Idea 2

- repo/package name: crackmapexecx
- bin short name cmex

###### Idea 3

- repo name: crackmapexec reloaded
- package name: crackmapexec
- bin short name cme

###### Idea 4

- repo name: crackmapexec community fork
- package name: crackmapexec
- bin short name cme

### **Not** using CrackMapExec name

###### Idea 5

- repo/package name: **NetWinUtils**
- bin short name **nwu**

###### Idea 6

- repo/package name: **WinMap**
- bin short name **winm**

###### Idea 7

- repo/package name: **ADMapExec**
- bin short name **ame**

###### Idea 8

- repo/package name: **SecNetMap**
- bin short name **snm**

###### Idea 9

- repo/package name: **ADAssault** (My fav ❤️)
- bin short name **ada**

###### Idea 10

- repo/package name: **BacktrackMapper**
- bin short name **btm**

###### Idea 11

- repo name: **aimsec**
- package name: **aim**
- meaning: **Active directory Infrastructure Mapper**
- bin short name **aim**

###### Idea 12

- repo/package name: **waat**
- meaning: **Windows Active directory Assessment Tool**
- bin short name **waat**

---

KrackMapExec?

>  (Although I would steer away from phonetically-similar names, since that doesn't sufficiently disambiguate the project when speaking about it, only writing.)
>
> Ref. https://github.com/Porchetta-Industries/CrackMapExec/discussions/801#discussioncomment-6957953

CarCakeMex
PamCreamCake?

Those are too funny and don't reflect the legacy of the project, people need to know just by reading the name that it's the continuation of crackmapexec.

**Reply by Marshall-Hallenbeck** [reply]
After the discussion with Marcello I think we are going to stay away from the "CrackMapExec" name, as much as we'd like to continue on its legacy.

**Reply by noraj** [reply]
@Marshall-Hallenbeck I have edited my post to suggest more names without re-using cme.

**Reply by bongobongoland** [reply]
@noraj 
dude, did you use ChatGPT to get generate those names? :)

**Reply by noraj** [reply]
No just my mind, some sound really bad tho.

All AI wanted to name it Network Mapper Pro.

**Reply by 0xswitch** [reply]
Unfortunately, ADAssault is too close to the french military vendor group "Dassault".  It could confuse people when pronouncing the name.

**Comment by bongobongoland** [comment]
I agree that Krack sounds similar to Crack; good point.
As for Pam Creamcake... that's was me joking... Surely you don't think I'd suggest Pam Creamcake as the new name for CME? :)

**Reply by noraj** [reply]
Who knows 😂

**Comment by bongobongoland** [comment]
Shall we call it... ByteBleeder? :))

**Comment by bongobongoland** [comment]
Ok peeps,

Good news,

I see some white smoke above the chimney,

We have a new name,

It's official,

It's here,

It has arrived,

The wait is over,

Get ready for this,

Expect the unexpected,

Hold on to your seatbelts....

Boom!

**DrShiva** (or DrS) is the new name.

Shocking! Outrageous! I couldn't believe it myself at first when I saw it. It's a stupid name. What were they thinking!?! But, apparently, that's the new name. Oh lawd, we have to live with it :( There's no going back. We're so screwed. What are we going to do!?

Wait. Give me a second. Ok. I've checked the official booklet which comes with the name. And it says the logic (there's no logic as far as I'm concerned) stems from the idea that there's Kali Linux and now there's Shiva. Very smart... Just like Kali, Shiva has multiple arms capable of holding multiple items (rdp, wmi, smb, ssh etc). But why is it a Doctor? I have no idea. Nothing makes sense anymore. Oh wait, let me turn the page of that booklet. OMG! Apparently, it's a pathologist, a medical examiner, who is dissecting, vivisecting, quartering, gutting, disembowelling, butchering the network, the DC (its guts... I mean the ntds.dit is hanging out!), the AD, its hosts and the admins (poor bastards, they all got slaughtered like lambs!!) with some crazy creepy medical instruments! I can't continue reading the booklet. I feel nauseous. It's too graphic, gory, morbid! It's ugly and dark... And its first version is v0.666!

Unholy!!!!!!1
Help us, God!!!!!1

**Reply by bongobongoland** [reply]
Shall I start working on a new logo? ;-P```

**Comment by Euzebius** [comment]
ADAssault / ada let's gooooo <3

**Comment by Marshall-Hallenbeck** [comment]
The developers have agreed on a name (after much brain wracking). We are in the process of renaming and fixing all the places in the code. Plus we'll need to get it added to apt/etc - please be patient during this migration period!
